pos-print/src/main/java/com/clouway/pos/print/PosPrintService.java

```diff
@@ -39,9 +40,13 @@ public static void main(String[] args) {
       new PersistentModule(client, commandCLI.dbName())
     );
 
+
     HttpBackend backend = new HttpBackend(commandCLI.httpPort(), injector);
     backend.start();
 
+    ReceiptPrintWorker backgroundPrinter = injector.getInstance(ReceiptPrintWorker.class);
+    backgroundPrinter.start();
+
     System.out.printf("POS Print Service is up and running on port: %d\n", commandCLI.httpPort());
 
     Runtime.getRuntime().addShutdownHook(new Thread(() -> {
```

Comment:
You have to shutdown printing service when termination signal is received.

Reply:
awaitTerminated() won't return, is it necessary for it to be called or should I just leave it as stopAsync()?